cmd/api/src/analysis/ad/queries.go

```diff
@@ -106,6 +106,11 @@ func GraphStats(ctx context.Context, db graph.Database) (model.ADDataQualityStat
 
 					for _, kind := range kinds {
 						innerKind := kind
+
+						if innerKind == ad.Entity {
```

Comment:
Yea it's to avoid running the count query for the `Base` kinds since that can be large and we wouldn't end up doing anything with it anyway. I think that's why the kind slice was originally not using the `NodeKinds` method so the continue effectively skips it.